tests/integrations/server/kv_service.rs

```diff
@@ -1744,3 +1746,207 @@ fn test_get_lock_wait_info_api() {
     must_kv_pessimistic_rollback(&client, ctx, b"a".to_vec(), 20);
     handle.join().unwrap();
 }
+
+#[test]
+fn test_api_version() {
+    const TIDB_KEY_CASE: &[u8] = b"t_a";
+    const TXN_KEY_CASE: &[u8] = &[key_prefix::TXN_KEY_PREFIX, 0, b'a'];
+    const RAW_KEY_CASE: &[u8] = &[key_prefix::RAW_KEY_PREFIX, 0, b'a'];
+
+    let test_data = vec![
+        // config api_version = V1|V1ttl, for backward compatible.
+        (ApiVersion::V1, ApiVersion::V1, TIDB_KEY_CASE, None),
+        (ApiVersion::V1, ApiVersion::V1, TXN_KEY_CASE, None),
+        // storage api_version = V1ttl, allow RawKV request only.
+        (
+            ApiVersion::V1ttl,
```

Comment:
We can set the storage API version as `ApiVersion::V1ttl`? @andylokandy

Reply:
Yes. Set `enable-ttl: true` and `api-version: 1` (or leave `api-version` as default)

*(codes [here](https://github.com/tikv/tikv/blob/1e1f32ab744cafd0145cd93e48249025455f6ae7/src/storage/config.rs#L121-L146))*
```
    pub fn api_version(&self) -> ApiVersion {
        match self.api_version {
            1 if self.enable_ttl => ApiVersion::V1ttl,
            1 => ApiVersion::V1,
            2 => ApiVersion::V2,
            _ => unreachable!(),
        }
    }

    pub fn set_api_version(&mut self, api_version: ApiVersion) {
        match api_version {
            ApiVersion::V1 => {
                self.api_version = 1;
                self.enable_ttl = false;
            }
            ApiVersion::V1ttl => {
                self.api_version = 1;
                self.enable_ttl = true;
            }
            ApiVersion::V2 => {
                self.api_version = 2;
                self.enable_ttl = false;
            }
        }
    }
```

Reply:
Oh, `ApiVersion::V2` should set  `self.enable_ttl = true;`! Would you like to correct it?